tests/tests/fv3_ccpp_gfs_v16beta_debug_RRTMGP

```diff
@@ -0,0 +1,85 @@
+###############################################################################
```

Comment:
The name of this file should be `tests/tests/fv3_ccpp_gfs_v16beta_RRTMGP_debug` and not `tests/tests/fv3_ccpp_gfs_v16beta_debug_RRTMGP`?